enterprise/tiny-comments.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Tiny Comments
+description: Tiny Comments provides the ability to add comments to the content and collaborate with other users for content editing.
+keywords: enterprise pricing video youtube vimeo mp3 mp4 mov movie clip film comment commenting mediaembed media
+---
+
+The [Tiny Comment]({{ site.baseurl }}/plugins/tiny-comment/) plugin provides the ability to add comments to the content and collaborate with other users for content editing within the TinyMCE editor.
+The primary value that TinyComments offer is a user interface to create, reply, and delete comments. The TinyComments plugin is built upon the the new [Annotations API]({{ site.baseurl }}/plugins/annotations/) and uses annotations to create comment threads.
+
+> Note: The TinyComments plugin in its current state does not store the comments anywhere. However, it gives the integrator callback APIs to create, search, and store comment threads.
+
+Example:
+
+```js
+    tinymce.init({
+       selector: 'textarea',
+       toolbar: 'bold italic underline | tinycomments',
+       plugins: 'tinycomments',
+       content_style: '.mce-annotation { background: yellow; color: black; } .tc-active-annotation {background: lime; color: black; }',
```

Comment:
@ephox-mogran This API looks strange to me. Why are all the "commenting" capabilities labelled as "annotations"? What if the user has a different annotation handler? 

I would have expected the commenting plugin to consume the "annotations" api, and produce a "commenting" api.